pupil-api/src/App.ts

```diff
@@ -16,8 +16,9 @@ const config = require('./config')
 import authRoutes from './routes/auth'
 import pingRoute from './routes/ping'
 
+let logLevel = 'debug'
 if (process.env.NODE_ENV !== 'production') {
-  winston.level = 'debug'
+  logLevel = 'error'
```

Comment:
Is this not the other way around? If `NODE_ENV` is production, `logLevel` would be error, and debug if in dev?